.gitignore

```diff
@@ -1,4 +1,3 @@
-etc/db/h2db.mv.db
```

Comment:
This file was added here as this was meant to be a starter database with prepopulated data that we would not want to edit. There are efforts to move this to a script, could we keep this until then please? #152 for reference.